test/test_torch.py

```diff
@@ -5875,6 +5875,17 @@ def test_serialization_storage_slice(self):
         self.assertEqual(s2[0], 0)
         self.assertEqual(s1.data_ptr() + 4, s2.data_ptr())
 
+    def test_load_unicode_error_msg(self):
```

Comment:
This test can (mostly) be kept around to make sure `legacy_conv2d.pt` loads without error.